lib/hammer_cli_katello/capsule.rb

```diff
@@ -136,7 +108,9 @@ class SyncCommand < HammerCLIForemanTasks::AsyncCommand
       autoload_subcommands
     end
 
-    autoload_subcommands
+    HammerCLIForeman::SmartProxy.subcommand("content",
```

Comment:
One more thing. Could we use ``HammerCLIKatello::Capsule::Content.command_name`` here rather than ``"content"``? Same reason that I asked about using ``#desc()``, it's one place to keep track of the command name, rather than two.